docs/spec.md

```diff
@@ -627,16 +627,16 @@ Performs element-wise addition of two tensors `lhs` and `rhs` and produces a
 
 #### Inputs
 
-| Label | Name  | Type                                  | Constraints |
-|-------|-------|---------------------------------------|-------------|
-| (I1)  | `lhs` | tensor or per-tensor quantized tensor | (C1)        |
-| (I2)  | `rhs` | tensor or per-tensor quantized tensor | (C1)        |
+| Label | Name  | Type                       | Constraints |
+|-------|-------|----------------------------|-------------|
+| (I1)  | `lhs` | tensor or quantized tensor | (C1)        |
+| (I2)  | `rhs` | tensor or quantized tensor | (C1)        |
```

Comment:
Add a constraints?

* `(is_per_axis_quantized(lhs) or is_per_axis_quantized(rhs)) == is_per_axis_quantized(result)`
* If `is_per_axis_quantized(lhs)`, then `quantization_dimension(lhs) == quantization_dimension(result)`
* If `is_per_axis_quantized(rhs)`, then `quantization_dimension(rhs) == quantization_dimension(result)`

Reply:
done. 
For record, here is the motivation for the suggetsion https://github.com/openxla/stablehlo/pull/1446#discussion_r1283425049

Reply:
Do we allow adding per-axis quantized tensor and per-tensor quantized tensor? 
By C1, I think such cases would be rejected by having different quantization_dimension.

Also, curious about the motivation to allow operand with different quantization parameters.

Reply:
@doyeonkim0 Thank you for pointing this out.  After reading the definition of `baseline_type` (i.e. `quantization_dimension` is guaranteed to be equal) again, I agree that `baseline_type(lhs) == baseline_type(rhs) == baseline_type(result)` can be sufficient for per-axis bias addition use case (which essentially guarantees that `lhs`, `rhs`, and `result` uses the same quantization dimension, scales, and zero points).

Reply:
@doyeonkim0 Thanks for pointing it point. I have updated the spec addressing your concern. The `baseline_type` should be used only when all the operands and results under comparison are non-quantized or per-tensor quantized.  

Yes, we do allow a mix of per-tensor and per-axis cases. 

>> Also, curious about the motivation to allow operand with different quantization parameters.
This is more of a flexibly added to almost all the quantized operations to have different quantization parameters.  This was possible mostly because the dq-op-q semantics for these ops.